### ISSUE TITLE
any plans for a release?

### ISSUE THREAD

**Issue by nafergo** [simeonradivoev/electron-dam#1]
This look great :) any plans for a release or blend support?

**Comment by simeonradivoev** [comment]
not really, pretty much abandoned. Lost interest. Might pick it up again once I again become too annoyed with having to deal with all the hassle of managing game assets.

**Comment by simeonradivoev** [comment]
ok I made a build, it still has lots of bugs, but it is still useful.
